pkg/services/licensing/oss.go

```diff
@@ -38,16 +38,16 @@ func (l *OSSLicensingService) LicenseURL(user *models.SignedInUser) string {
 
 func (l *OSSLicensingService) Init() error {
 	l.HooksService.AddIndexDataHook(func(indexData *dtos.IndexViewData, req *models.ReqContext) {
-		for _, node := range indexData.NavTree {
-			if node.Id == "admin" {
-				node.Children = append(node.Children, &dtos.NavLink{
-					Text: "Upgrade",
-					Id:   "upgrading",
-					Url:  l.LicenseURL(req.SignedInUser),
-					Icon: "unlock",
-				})
-			}
-		}
+		// for _, node := range indexData.NavTree {
+		// 	if node.Id == "admin" {
+		// 		node.Children = append(node.Children, &dtos.NavLink{
+		// 			Text: "Upgrade",
+		// 			Id:   "upgrading",
+		// 			Url:  l.LicenseURL(req.SignedInUser),
+		// 			Icon: "unlock",
+		// 		})
+		// 	}
+		// }
```

Comment:
This comment should be removed altogether

Reply:
Done